docs/index.rst

```diff
@@ -0,0 +1,51 @@
+.. toctree::
```

Comment:
This is pretty readable, but did you consider using markdown via restructify as documented here:
http://recommonmark.readthedocs.io/en/latest/auto_structify.html